TempleApp/src/main/java/example/TempleApp/activities/ReadSingleData.java

```diff
@@ -101,27 +101,27 @@ protected void onPreExecute() {
             super.onPreExecute();
 
             dialog = new ProgressDialog(ReadSingleData.this);
-            dialog.setTitle(getString(R.string.wait));
-            dialog.setMessage(getString(R.string.fetch));
+            dialog.setTitle(getString(example.TempleApp.R.string.wait));
+            dialog.setMessage(getString(example.TempleApp.R.string.fetching));
             dialog.show();
 
         }
 
         @Nullable
         @Override
         protected Void doInBackground(Void... params) {
-            Log.i(Controller.TAG, getString(R.string.ID_value) + id);
+            Log.i(Controller.TAG, getString(example.TempleApp.R.string.ID_value) + id);
             JSONObject jsonObject = Controller.readData(id);
-            Log.i(Controller.TAG, getString(R.string.json_obj) + jsonObject);
+            Log.i(Controller.TAG, getString(example.TempleApp.R.string.Json_obj) + jsonObject);
 
             try {
                 /*
                   Check Whether Its NULL???
                  */
                 if (jsonObject != null) {
 
-                    JSONObject user = jsonObject.getJSONObject(getString(R.string.user));
-                    name = user.getString(getString(R.string.name_));
+                    JSONObject user = jsonObject.getJSONObject(getString(example.TempleApp.R.string.user));
```

Comment:
all the `example.TempleApp.R.string` are not necessary. `R.string` can be used.

revert all these kind